packages/verified-fetch/src/verified-fetch.ts

```diff
@@ -294,6 +324,9 @@ export class VerifiedFetch {
       if (options?.signal?.aborted === true) {
         throw new AbortError('signal aborted by user')
       }
+      if (['ERR_NO_PROP', 'NO_TERMINAL_ELEMENT'].includes(err.code)) {
```

Comment:
https://github.com/ipfs/helia-verified-fetch/pull/39/files#r1553442086

```suggestion
      if (['ERR_NO_PROP', 'ERR_NO_TERMINAL_ELEMENT'].includes(err.code)) {
```